src/ol_infrastructure/applications/airbyte/Pulumi.airbyte.applications.QA.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
The naming convention for the stacks is `applications.airbyte.QA` so that everything is namespaced under `applications`, `infrastructure`, etc.